services/WorkflowService.ts

```
@@ -0,0 +1,174 @@
+      );
+      return response.data.workflow;
+    } catch (error) {
+      console.error("Error updating workflow:", error);
+      throw error;
+    }
+  }
+
+  static async deleteWorkflow(workflowId: string): Promise<boolean> {
+    try {
+      const headers = await getHeaders();
+      await axios.delete(`${this.BASE_URL}/${workflowId}`, { headers });
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)